pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tankersdk-identity"
-version = "1.2.1"
+version = "1.3.0"
```

Comment:
1.3.1, no ?

Reply:
1.3.0 is the latest released, 1.3.1 is the future, I don't know what's the better option?

Reply:
never mind. Keep 1.3.1